sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -217,4 +217,6 @@ void saveBulkEditWithFacilities(
 	List<CaseDataDto> getByExternalId(String externalId);
 
 	void updateExternalData(List<ExternalDataDto> externalData) throws ExternalDataUpdateException;
+
+	void updateCompletenessTask();
```

Comment:
That was the signature required: `CaseFacade.updateCompleteness:int` with number of cases updated as result

Reply:
updated